packages/framework/use-tokens/src/__snapshots__/useTokens.samples.test.tsx.snap

```diff
@@ -0,0 +1,163 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`useTokens samples Sample1Text rendering with no overrides 1`] = `
+<SampleText1>
+  <Text
+    style={
+      Object {
+        "color": "black",
+        "fontFamily": "Arial",
+        "fontSize": 12,
+      }
+    }
+  >
+    <Text
```

Comment:
I'm not understanding why there's identical Text elements and one is nested within the other. Can you explain where the inner element is coming from?

Reply:
Sorry was on vacation, I believe the inner text element is the equivalent of the native component when doing a full mounting. i.e. an implementation detail of the testing framework.

Reply:
Thanks for the explanation, makes sense.